appverifier/src/main/java/com/android/mdl/appreader/MainActivity.kt

```diff
@@ -43,6 +43,11 @@ class MainActivity : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
 
         mAdapter = NfcAdapter.getDefaultAdapter(this)
+        mAdapter?.enableReaderMode(
+            this, null,
+            NfcAdapter.FLAG_READER_NFC_A + NfcAdapter.FLAG_READER_NFC_B
+                    + NfcAdapter.FLAG_READER_SKIP_NDEF_CHECK + NfcAdapter.FLAG_READER_NO_PLATFORM_SOUNDS,
+            null)
```

Comment:
Is there a reason why we're enabling readermode here as well? Isn't it enough to do it in TransferManager.kt?

Reply:
I noticed if you only add the flag via `enableReaderMode` in TransferManager.kt, then the NfcAdapter would naturally make the sound to indicate a tag was detected in the cases where the user hadn't yet move to the Transfer fragment (by clicking the button). This meant that you could still get a notification noise for detected tags in 2 scenarios: (1) upon opening the app, on the first screen with the options, before moving to the Transfer fragment and (2) when the app was simply installed on the phone and not running, but a phone with the holder app touches the verifier phone. 

I added this `enableReaderMode` in MainActivity.kt to address those problems with the thought that the Transfer fragment could simply overwrite the callback when it was active/needed.

Reply:
Looks to me like the root problem simply is that we make the user have to press "Next" to actually do what the application was designed to do, which is annoying. I filed Issue #252 to fix that so we can start reader mode early on.

So, basically, I don't think we should enable readerMode in `MainActivity.kt` right now... let's wait with that until we work on #252.

Reply:
I fully agree. To clarify, should I remove the `enableReaderMode` in `MainActivity.kt` to merge this PR now, or should I hold off until #252 is complete?

Reply:
Just remove the code in `MainActivity.kt` for now so we can land this.